src/Mono.Android/Microsoft.Android.Runtime/RuntimeFeature.cs

```diff
@@ -5,9 +5,21 @@ namespace Microsoft.Android.Runtime;
 
 static class RuntimeFeature
 {
+	const bool ManagedTypeMapEnabledByDefault = false;
+	const bool IsMonoRuntimeEnabledByDefault = true;
+	const bool IsCoreClrRuntimeEnabledByDefault = false;
+
 	const string FeatureSwitchPrefix = "Microsoft.Android.Runtime.RuntimeFeature.";
 
 	[FeatureSwitchDefinition ($"{FeatureSwitchPrefix}{nameof (ManagedTypeMap)}")]
 	internal static bool ManagedTypeMap { get; } =
-		AppContext.TryGetSwitch ($"{FeatureSwitchPrefix}{nameof (ManagedTypeMap)}", out bool isEnabled) ? isEnabled : false;
+		AppContext.TryGetSwitch ($"{FeatureSwitchPrefix}{nameof (ManagedTypeMap)}", out bool isEnabled) ? isEnabled : ManagedTypeMapEnabledByDefault;
+
+	[FeatureSwitchDefinition ($"{FeatureSwitchPrefix}{nameof (IsMonoRuntime)}")]
+	internal static bool IsMonoRuntime { get; } =
+		AppContext.TryGetSwitch ($"{FeatureSwitchPrefix}{nameof (IsMonoRuntime)}", out bool isEnabled) ? isEnabled : IsMonoRuntimeEnabledByDefault;
```

Comment:
I wonder if we should enforce that the feature switch is always set explicitly and throw if it isn't. I'm worried that we could introduce a regression and for example we would drop the `Microsoft.Android.Runtime.RuntimeFeature.IsMonoRuntime` on CoreCLR in the SDK and now both `IsMonoRuntime` and `IsCoreClrRuntime` would be true.

Reply:
I'll add something that throws on startup if both are set to true.